packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -2683,7 +2691,7 @@ class StackViewport extends Viewport {
     });
   }
 
-  private resetCameraGPU({ resetPan, resetZoom }): boolean {
+  private resetCameraGPU({ resetPan, resetZoom, resetAspectRatio }): boolean {
```

Comment:
Can you just pass the reset options to the parent rather than needing to specifically list them?  It means that all new reset options will just flow through unless specifically editted.
eg:
resetCameraGPU(resetOptions?: ResetOptions): boolean

...
return super.resetCamera({
   resetToCenter,
   ...resetOptions
})